### PR TITLE
Add Create DashApp and create Singlestore Cloud Functions template to spaces 

### PR DESCRIPTION
**Pull request by bharathts07**
WHAT 

1. Add a notebook to demonstrate how to create a simple Singlestore Cloud function
2. Add a notebook to demonstrate how to create a simple DashApp

WHY

1. New features planned to be made GA soon.

### CODE REVIEW

File: notebooks/cloud-functions-template/notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+        "import requests\n",
+        "\n",
+        "ca_cert_url = \"https://portal.singlestore.com/static/ca/singlestore_bundle.pem\"\n",
+        "ca_cert_path = \"/tmp/singlestore_bundle.pem\"\n",
+        "\n",
+        "response = requests.get(ca_cert_url)\n",
+        "with open(ca_cert_path, \"wb\") as f:\n",
+        "    f.write(response.content)\n",
+        "\n",
+        "sql_connection_string = connection_url.replace(\"singlestoredb\", \"mysql+pymysql\")\n",
```
Comment (kesmit13) [inline notebooks/cloud-functions-template/notebook.ipynb:99]: Why are you pulling out the singlestoredb driver and using pymysql?

File: notebooks/create-dash-app/notebook.ipynb [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+        "import requests\n",
+        "\n",
+        "ca_cert_url = \"https://portal.singlestore.com/static/ca/singlestore_bundle.pem\"\n",
+        "ca_cert_path = \"/tmp/singlestore_bundle.pem\"\n",
+        "\n",
+        "response = requests.get(ca_cert_url)\n",
+        "with open(ca_cert_path, \"wb\") as f:\n",
+        "    f.write(response.content)\n",
+        "\n",
+        "sql_connection_string = connection_url.replace(\"singlestoredb\", \"mysql+pymysql\")\n",
```
Comment (kesmit13) [inline notebooks/create-dash-app/notebook.ipynb:142]: Same question as above.

File: notebooks/cloud-functions-template/notebook.ipynb (hunk 1 above)
Comment (srinathnarayanan) [reply notebooks/cloud-functions-template/notebook.ipynb:99]: This is a demo for dashboards and cloud functions.
%%sql doesnt work in a multi threaded manner, which is why we need to use the sqlalchemy engine with a connection pool for now.

File: notebooks/create-dash-app/notebook.ipynb (hunk 2 above)
Comment (srinathnarayanan) [reply notebooks/create-dash-app/notebook.ipynb:142]: replied above

File: notebooks/cloud-functions-template/notebook.ipynb (hunk 1 above)
Comment (kesmit13) [reply notebooks/cloud-functions-template/notebook.ipynb:99]: I still don't understand. Why doesn't SinlgeStoreDB work? PyMySQL has the exact same thread issues that the SingleStoreDB client does.

**Review by srinathnarayanan — APPROVED** [review]

**Comment by bharathts07** [comment]
I will merge and test this

**Pull request merged**